src/components/NavItem/NavItem.tsx

```diff
@@ -0,0 +1,129 @@
+/* External dependencies */
+import React, { useCallback, useMemo, memo } from 'react'
+import { noop, isNil } from 'lodash-es'
+
+/* Internal dependencies */
+import useDepthContext, { DepthContext } from 'Hooks/useDepthContext'
+import { IconSize, isIconName } from 'Components/Icon'
+import type NavItemProps from './NavItem.types'
+import {
+  ItemWrapper,
+  LeftIconWrapper,
+  StyledIcon,
+  ChevronWrapper,
+  ContentWrapper,
+  RightContentWrapper,
+} from './NavItem.styled'
+
+const DEFAULT_INDENT = 30
+const DEFAULT_PADDING_LEFT = 6
+export const NAV_ITEM_TEST_ID = 'bezier-react-nav-item'
+
+function NavItem(
+  {
+    as,
+    testId = NAV_ITEM_TEST_ID,
+    name,
+    style,
+    className,
+    interpolation,
+    children,
+    content,
+    rightContent,
+    leftIcon,
+    leftIconColor = 'txt-black-dark',
+    indent = DEFAULT_INDENT,
+    active,
+    open,
+    onClick = noop,
+  }: NavItemProps,
+) {
```

Comment:
```suggestion
function NavItem({
  as,
  testId = NAV_ITEM_TEST_ID,
  name,
  style,
  className,
  interpolation,
  children,
  content,
  rightContent,
  leftIcon,
  leftIconColor = 'txt-black-dark',
  indent = DEFAULT_INDENT,
  active,
  open,
  onClick = noop,
}: NavItemProps) {
```

`forwardRef`를 적용할 때에는 `(`와 `{`를 newline으로 분리하지만, 그렇지 않다면 분리하지 않습니당